cluster-autoscaler/main.go

```diff
@@ -408,6 +409,8 @@ func getKubeConfig() *rest.Config {
 		klog.Fatalf("Failed to build Kubernetes client configuration: %v", err)
 	}
 
+	kubeConfig.ContentType = *kubeAPIContentType
```

Comment:
I understand `ContentType` is empty now, but it could change in the future, so maybe let's only use the flag value if it was actually provided?

Reply:
I do like to keep the consistency for CA with other k8s components in using this flag. 

`--kube-api-content-type string     Default: application/vnd.kubernetes.protobuf`

(eg. https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)

If it ever changes, maybe change this part in the future? alternatively we can specify a default value in `kubeAPIContentType`.

Reply:
Done, thanks!